AppSearchSample/app/src/main/java/com/android/example/appsearchsample/NoteListItemAdapter.kt

```diff
@@ -62,19 +59,18 @@ class NoteListItemAdapter(private val onDelete: (SearchResult?) -> Unit) :
     private val noteTextView = binding.noteText
     private val noteDeleteButtonView = binding.noteDeleteButton
 
-    fun bind(sr: SearchResult, onDelete: (SearchResult?) -> Unit, position: Int) {
-      val note = sr.genericDocument.toDocumentClass(Note::class.java)
-      val sb = SpannableStringBuilder(note.text)
+    fun bind(searchResult: SearchResult, onDelete: (SearchResult?) -> Unit) {
+      val note = searchResult.genericDocument.toDocumentClass(Note::class.java)
+      val stringBuilder = SpannableStringBuilder(note.text)
 
-      sr.matchInfos.forEach {
-        sb.setSpan(StyleSpan(BOLD), it.exactMatchRange.start, it.exactMatchRange.end, Spannable.SPAN_INCLUSIVE_INCLUSIVE)
+      searchResult.matchInfos.forEach {
+        if(it.propertyPath == "text")
```

Comment:
Instead of hardcoding path, please add this path as a constant within the companion object of the class.